### PR TITLE
feat(Ripple): hide effect for touch scroll events (#7887)

### PR DESCRIPTION
**Pull request by hareku**
## Description

Don't display the ripple when mobile users scroll ripple elements.

## Motivation and Context

for https://github.com/vuetifyjs/vuetify/issues/7887

## How Has This Been Tested?

playground (visually)

## Markup:

<details>

```vue
<template>
  <v-container>
    <h1>Cards</h1>
    <v-card ripple v-for="key in 5" :key="key" class="mb-3">
      <v-card-title>Hello</v-card-title>
    </v-card>

    <v-divider class="my-8" />

    <h1>List</h1>
    <v-list>
      <v-list-item v-for="key in 5" :key="key" @click="() => {}">
        <v-list-item-content>Hello</v-list-item-content>
      </v-list-item>
    </v-list>
  </v-container>
</template>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/directives/ripple/index.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import './VRipple.sass'
 import { VNode, VNodeDirective } from 'vue'
 import { consoleWarn } from '../../util/console'
 
+const DELAY_RIPPLE = 80
```
Comment (hareku) [inline packages/vuetify/src/directives/ripple/index.ts]: refer to material-ui
https://github.com/mui-org/material-ui/pull/8605/files#diff-6126a8f8604be55b8d82367154c85852R11

**Comment by hareku** [comment]
@KaelWD 
Would you please review this PR?

**Comment by Sshuichi** [comment]
Can we have a review here please?  <3

**Comment by Tantawi** [comment]
@KaelWD Please it is kind of important for UX.

**Comment by yoshschoemaker** [comment]
@KaelWD please take a look at this, it's a very important fix.

**Comment by Sshuichi** [comment]
Fuck yes!!

**Pull request merged**